setup/paths.sh

```diff
@@ -37,38 +37,27 @@ else
     source $BUILD_HOME/cmsgemos/setup/paths.sh
 fi
 
+# Export project
+export GEM_PLOTTING_PROJECT=$BUILD_HOME/gem-plotting-tools
+
+# Setup Path
+export PATH=$PATH:$GEM_PLOTTING_PROJECT
+export PATH=$PATH:$GEM_PLOTTING_PROJECT/macros
+export PATH=$PATH:$GEM_PLOTTING_PROJECT/setup
+
+# Setup PYTHONPATH
+export PYTHONPATH=$PYTHONPATH:$BUILD_HOME/vfatqc-python-scripts
+export PYTHONPATH=$PYTHONPATH:$GEM_PLOTTING_PROJECT
+export PYTHONPATH=$PYTHONPATH:$GEM_PLOTTING_PROJECT/macros
+export PYTHONPATH=$PYTHONPATH:$GEM_PLOTTING_PROJECT/setup
```

Comment:
Then I suppose it's time that project be organized more like a proper `python` package.
* `macros` has a nebulously defined name, but should be reserved for a piece of code you're actually going to run a la: `python mymacro.py`; if the module is being imported or loaded elsewhere, it's better to structure the package like a proper library, i.e., 
 ```
gem-plotting-tools/
  - main program files (probably expected to be executable and in the $PATH, could also go in a bin subdirectory)
  macros/
    - tools you may use for various tasks on their own
  fitting/
   - modules related to fitting
  mapping/
   - modules related to mapping
  tests/
    - any unit tests
 ```
* This way you can `import mapping.PanChannelMaps` (or `from mapping.PanChannelMaps import *`, or however you are using them) with only the `gem-plotting-tools` directory having been added to the `$PYTHONPATH`